README.md

```diff
@@ -22,7 +22,33 @@ Alternatively you can install with Golang > 1.18 with:
 go install github.com/openaq/openaq-cli
 ```
 
```

Comment:
Lets move the Homebrew and Scoop installation methods above the Github release and `go install` methods, those should be later and more targeted at advanced users